Jenkinsfile

```diff
@@ -1,44 +1,217 @@
+                                      conda create -n isis python=3
+                                    '''
+
+                                    if (lower_label == "centos") {
+                                      sh 'conda env update -n isis -f environment_gcc4.yml --prune'
+                                    } else {
+                                      sh 'conda env update -n isis -f environment.yml --prune'
+                                    }
+
+                                    withEnv(isisEnv) {
+                                        dir("${env.ISISROOT}") {
+                                            try {
+                                                env.STAGE_STATUS = "Building ISIS on ${label}"
+                                                sh """
+                                                    source activate isis
+                                                    cmake -GNinja ${cmakeFlags.join(' ')} ../isis
+                                                    ninja -j4 install
+                                                    python ../isis/scripts/isis3VarInit.py --data-dir ${env.ISIS3DATA} --test-dir ${env.ISIS3TESTDATA}
+                                                """
+                                            }
+                                            catch(e) {
+                                                build_ok = false
+                                                errors.add(env.STAGE_STATUS)
+                                                println e.toString()
+                                            }
+                                        }
+
+                                        if (build_ok) {
+                                            dir("${env.ISISROOT}") {
+                                                try  {
+                                                    env.STAGE_STATUS = "Running unit tests on ${env.OS}"
+                                                        sh """
+                                                            source activate isis
+                                                            echo $ISIS3TESTDATA
+                                                            echo $ISIS3DATA
+
+                                                            # environment variables
+                                                            export ISISROOT=${env.ISISROOT}
+                                                            export ISIS3TESTDATA="/isisData/testData"
+                                                            export ISIS3DATA="/isisData/data"
+                                                            export PATH=`pwd`/../install/bin:/home/jenkins/.conda/envs/isis/bin:$PATH
+
+                                                            automos -HELP
+                                                            catlab -HELP
+                                                            tabledump -HELP
```

Comment:
What are these here for?

Reply:
They were originally there for checking if we had access to the isis apps in the path since jenkins doesn't play well with modifying environment variables like $PATH. They could probably be removed but they act as a nice failure state that avoids running all of the tests.